static/gen.php

```diff
@@ -0,0 +1,107 @@
+<?php
+die('disabled');
+
+if (!is_writable(__DIR__)) {
+    http_response_code(500);
+    exit('Directory is not writable.');
```

Comment:
i would prefer if `exit` is called with integer parameter, and `die` for textual parameters. they are aliases in php, so this is just personal preference